src/fft.jl

```diff
@@ -578,6 +578,9 @@ end
 
 # low-level FFTWPlan creation (for internal use in FFTW module)
 
+_mapIntknd(knd, region, kinds) = map(Int, knd)
+_mapIntknd(knd, region::Union{AbstractUnitRange,Tuple}, kinds::Integer) = ntuple(x->Int(kinds), length(region))
```

Comment:
If `region` is a tuple then we could have
```jl
_mapIntknd(knd, region::NTuple{N}, kinds::Integer) where {N} = ntuple(x->Int(kinds), Val{N}())
```
to ensure that the compiler knows about the length.  But I guess constant-propagation is good enough these days for `length(region)` to propagate into `ntuple`?